### PR TITLE
Misc cleanup changes for the 1.1 release

### PR DESCRIPTION
**Pull request by MichaelSimons**
@naamunds

### CODE REVIEW

File: 1.0/README.md [hunk 1]
```diff
@@ -203,6 +203,8 @@ It contains the .NET Core SDK which is comprised of two parts:
 
 Use this image for your development process (developing, building and testing applications).
 
+There currently are two flavors of the sdk images, `projectjson` and `msbuild`.  These two flavors exist while the [transition occurs from project.json to msbuild](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/).  Once the tooling stabilizes the project json variant will be deprecated and there will only be a msbuild variant.
```
Comment (naamunds) [inline 1.0/README.md]: "an msbuild variant"

Comment (naamunds) [inline 1.0/README.md]: Consider adding a comma after "stabilizes".

File: test/create-run-publish-app.sh [hunk 2]
```diff
@@ -9,14 +9,8 @@ cd $1
...
-if [[ $2 == "1.1"* ]]; then
-    replacement_arg="s/1.0.1/1.1.0/"
-
-    if [[ $2 == *"projectjson"* ]]; then
-        sed -i ${replacement_arg} ./project.json
-    else
-        sed -i ${replacement_arg} ./${PWD##*/}.csproj
-    fi
+if [[ $2 == "1.1-sdk-msbuild" ]]; then
+    sed -i "s/1.0.1/1.1.0/;s/netcoreapp1.0/netcoreapp1.1/" ./${PWD##*/}.csproj
```
Comment (naamunds) [inline test/create-run-publish-app.sh]: Why do we no longer need the logic for replacing 1.0.1 with 1.1.0 in the project.json?  Does preview2.1 now use 1.1.0 by default whereas it previously used 1.0.1?

File: 1.0/README.md (hunk 1 above)
Comment (MichaelSimons) [inline 1.0/README.md]: Fixed

Comment (MichaelSimons) [inline 1.0/README.md]: Fixed

File: test/create-run-publish-app.sh (hunk 2 above)
Comment (MichaelSimons) [inline test/create-run-publish-app.sh]: Preview2.1 is based on 1.1.0 therefore running this logic does nothing except add overhead.

**Pull request merged**